tests/runtime/configs.yaml

```diff
@@ -0,0 +1,12 @@
+---
+- name:     Device1
+  kindName: CPU
```

Comment:
minor nit, if we are going to use a CPU maybe rename the parameters so it's clear they are test parameters, or put a comment that these parameters don't mean anything to the CPU backend.